node/src/chain_specs/dolphin.rs

```diff
@@ -0,0 +1,231 @@
+		Extensions {
+			relay_chain: "".into(),
+			para_id: CALAMARI_PARACHAIN_ID.into(),
+		},
+	)
+}
+
+fn dolphin_dev_genesis(
+	invulnerables: Vec<(AccountId, AuraId)>,
+	root_key: AccountId,
+	endowed_accounts: Vec<AccountId>,
+) -> dolphin_runtime::GenesisConfig {
+	dolphin_runtime::GenesisConfig {
+		system: dolphin_runtime::SystemConfig {
+			code: dolphin_runtime::WASM_BINARY
+				.expect("WASM binary was not build, please build it!")
+				.to_vec(),
+		},
+		balances: dolphin_runtime::BalancesConfig {
+			balances: endowed_accounts[..endowed_accounts.len() / 2]
+				.iter()
+				.map(|k| {
+					(
+						k.clone(),
+						100 * CALAMARI_ENDOWMENT / ((endowed_accounts.len() / 2) as Balance),
+					)
+				})
+				.collect(),
+		},
+		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
+		// of this.
+		aura: Default::default(),
+		sudo: dolphin_runtime::SudoConfig { key: root_key },
+		parachain_info: dolphin_runtime::ParachainInfoConfig {
+			parachain_id: CALAMARI_PARACHAIN_ID.into(),
+		},
+		collator_selection: dolphin_runtime::CollatorSelectionConfig {
+			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+			candidacy_bond: DOL * 1000, // How many tokens will be reserved as collator
+			..Default::default()
+		},
+		session: dolphin_runtime::SessionConfig {
+			keys: invulnerables
+				.iter()
+				.cloned()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                 // account id
+						acc,                         // validator id
+						dolphin_session_keys(aura), // session keys
+					)
+				})
+				.collect(),
+		},
+		democracy: DemocracyConfig::default(),
+		council: CouncilConfig {
+			members: endowed_accounts.iter().take(1).cloned().collect(),
+			phantom: Default::default(),
+		},
+		technical_committee: TechnicalCommitteeConfig {
+			members: endowed_accounts.iter().take(1).cloned().collect(),
+			phantom: Default::default(),
+		},
+		council_membership: Default::default(),
+		technical_membership: Default::default(),
+		aura_ext: Default::default(),
+		parachain_system: Default::default(),
+		polkadot_xcm: dolphin_runtime::PolkadotXcmConfig {
+			safe_xcm_version: Some(0),
+		},
+	}
+}
+
+// pub fn dolphin_testnet_config() -> Result<DolphinChainSpec, String> {
+// 	let mut spec = DolphinChainSpec::from_json_bytes(
+// 		&include_bytes!("../../../genesis/dolphin-testnet-genesis.json")[..],
+// 	)?;
+// 	spec.extensions_mut().para_id = CALAMARI_PARACHAIN_ID.into();
+// 	Ok(spec)
+// }
+
+// // Dolphin testnet for ci jobs
+// pub fn dolphin_testnet_ci_config() -> Result<DolphinChainSpec, String> {
+// 	DolphinChainSpec::from_json_bytes(
+// 		&include_bytes!("../../../genesis/dolphin-testnet-ci-genesis.json")[..],
+// 	)
+// }
+
+// // Dolphin mainnet
+// pub fn dolphin_config() -> Result<DolphinChainSpec, String> {
+// 	DolphinChainSpec::from_json_bytes(
+// 		&include_bytes!("../../../genesis/dolphin-genesis.json")[..],
+// 	)
+// }
```

Comment:
Commented code.
And also please add any secrets in the password store.
Please ping Rob in order to add your gpg key to the master key.

Reply:
We don't need `dolphin_testnet_config` adn `dolphin_testnet_ci_config` for now but we should use the `dolphin_config()` one, the json is already generated.